src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
i am fine with trying to get it right. But i am not sure what/how to rebase at this point, could you elaborate a bit?